### ISSUE TITLE
Huawei push - Android 12 compatibility

### ISSUE THREAD

**Issue by nestorsgarzonc** [HMS-Core/hms-flutter-plugin#144]
**Description**
Recently I added android 12 compatibility to my flutter app and I added the huawei push plugin but when I try to run the app I get the following error but when I remove the plugin my app run correctly:

````
android:exported needs to be explicitly specified for <receiver>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
````

**Screenshots**
![image](https://user-images.githubusercontent.com/40438456/142934400-77578740-9dbf-4816-995b-68c7984c5400.png)
![image](https://user-images.githubusercontent.com/40438456/142934436-ba5f8a4b-6a6c-4c02-bd21-c40114bef9cf.png)
![image](https://user-images.githubusercontent.com/40438456/142934502-fafef7f1-b70b-4205-8da4-30bbee574cfc.png)

**Environment**
 - Platform: Flutter
 - Kit: Push
 - Kit Version: huawei_push: ^5.3.0+304
 - OS Version: Android 12
 - Your Location/Region: Colombia

**Comment by furkansarihan** [comment]
Hello @nestorsgarzonc,

This Flutter plugin is a wrapper around Native SDK and Huawei Push Kit SDK is not supports Android 12 yet. You may find related information on [this page](https://developer.huawei.com/consumer/en/doc/development/HMSCore-Guides/service-introduction-0000001050040060).

**Comment by RicharC293** [comment]
@furkansarihan  Is there an estimated time to support Android 12?

**Comment by SerdarCanDev** [comment]
Hi @RicharC293 for any questions related to Native SDKs please [submit a ticket online](https://developer.huawei.com/consumer/en/support/feedback/).

**Comment by RicharC293** [comment]
This was be solved with huawei_push: ^6.1.0+300
